akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/MaxThroughputSpec.scala

```diff
@@ -55,6 +54,12 @@ object MaxThroughputSpec extends MultiNodeConfig {
            # See akka-remote-tests/src/test/resources/aeron.properties
            #advanced.embedded-media-driver = off
            #advanced.aeron-dir = "target/aeron"
+           
+           #advanced.compression {
+           #  enabled = on
+           #  actor-refs.enabled = on
+           #  manifests.enabled = on
+           #}
          }
        }
```

Comment:
Ah, no that we don't have yet.
The perf flamegraphs actually do the right thing here, since they attach after 5 seconds, but the notmal measurements in the test don't (yet)